bug: fix LoginTest.cs bug

MartyOfMCA opened the pull request:
remove errors and write unit tests against the ResetFields and UserProvidedData for Login.cs source code

Commit:
bug: fix LoginTest.cs bug

remove errors and write unit tests against the ResetFields and
UserProvidedData for Login.cs source code

Commit:
refactor: change test function name

change the name for the TestUserProvidedData function to correct
the typo

denkyipatrick commented on test/FPIS_TEST/LoginTest.cs:
```diff
@@ -7,26 +7,45 @@ namespace FPIS_TEST
         }
 
         [TestMethod]
         public void TestAuthenticateUser()
```
What awesomeness are we about to see 🤣 ? 🙏 Please can you remove this method if it's not yet implemented?

Commit:
fix: remove empty test method

MartyOfMCA committed:
Merge branch 'develop' into fix-logintest

MartyOfMCA committed:
feat: add LoginService

add LoginService to handle all the login database manipulations

MartyOfMCA committed:
fix: add unit test to LoginService

add unit test cases to test for a successful login attempt
and a failed login attempt.

MartyOfMCA committed:
feat: do not add App.config file to project
